testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jaxrs/tracing/TracingFeatureTestCase.java

```diff
@@ -25,19 +25,28 @@ public class TracingFeatureTestCase {
     @Deployment
     public static Archive<?> deploy() {
         WebArchive war = ShrinkWrap.create(WebArchive.class, "tracing.war");
-        war.addClasses(TracingConfigResource.class);
+        war.addClasses(TracingConfigResource.class, TracingApp.class);
         war.addAsWebInfResource(WebXml.get(
-                "<servlet-mapping>\n"
-                        + "        <servlet-name>javax.ws.rs.core.Application</servlet-name>\n"
-                        + "        <url-pattern>/*</url-pattern>\n" + "</servlet-mapping>\n"
-                        + "<context-param>\n" +
-                        "    <param-name>resteasy.server.tracing.type</param-name>\n" +
-                        "    <param-value>ALL</param-value>\n" +
-                        "</context-param>\n" +
-                        "<context-param>\n" +
-                        "    <param-name>resteasy.server.tracing.threshold</param-name>\n" +
-                        "    <param-value>VERBOSE</param-value>\n" +
-                        "</context-param>\n"), "web.xml");
+                "    <context-param>\n" +
+                        "        <param-name>resteasy.server.tracing.type</param-name>\n" +
+                        "        <param-value>ALL</param-value>\n" +
+                        "    </context-param>\n" +
+                        "    <context-param>\n" +
+                        "        <param-name>resteasy.server.tracing.threshold</param-name>\n" +
+                        "        <param-value>VERBOSE</param-value>\n" +
+                        "    </context-param>\n" +
+                        "    <servlet>\n" +
```

Comment:
@jamezp I have to use servlet setting here for above `context-param` to work. I'm not sure whether this is the proper way. If there's better way please let me know.